.github/workflows/main.yaml

```diff
@@ -126,6 +138,7 @@ jobs:
         config: [devel, debug, qa, gold]
     container: ${{ needs.docker-build-image.outputs.docker_tag }}
     env:
+      TEST_ARTIFACTS_KEY: ${{ matrix.name }}_test_artifacts
```

Comment:
Let's change this to include platform name, otherwise test tar archive name collides between linux and linux-chrobalt platforms.  Something like so:

TEST_ARTIFACTS_KEY: ${{ matrix.platform }}_${{ matrix.name }}_test_artifacts